rxandroid/src/main/java/rx/android/lifecycle/LifecycleObservable.java

```diff
@@ -72,6 +77,30 @@ public Boolean call(LifecycleEvent lifecycleEvent) {
     }
 
     /**
+     * Similar to {@link #bindActivityLifecycle(Observable, Observable)} but uses
+     * {@link android.app.Application.ActivityLifecycleCallbacks} provided in
+     * API 14 to listen for Activity lifecycle.
+     *
+     * @param activity the activity we want to monitor lifecycle sequence for
+     * @param source   the source sequence
+     */
+
+    @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
+    public static <T> Observable<T> bindActivityLifecycle(Activity activity, Observable<T> source) {
+        // Make sure we're running on ICS or higher to use ActivityLifecycleCallbacks
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
+            throw new IllegalStateException ("This method is only available on API >= 14");
```

Comment:
I don't believe this library supports minSdkVer < 14 anyways. Could be wrong though.